templates/registration/login.html

```diff
@@ -0,0 +1,15 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>Login</h1>
+
+  {% if form.errors %}
+    <p style="color: red">Invalid credentials</p>
+  {% endif %}
+
+  <form method="post" action="{% url 'login' %}">
```

Comment:
The action attribute in the form tag should match the URL pattern name for the login view. Ensure that 'login' is the correct name for the login URL in your urls.py file.